python/setup.cfg

```diff
@@ -19,6 +19,11 @@ long_description_content_type = text/markdown
 keywords = 
 	differential privacy
 
+[project]
+dependencies = [
+	'deprecated'
```

Comment:
This seems to be the first runtime python dependency that we've added that isn't an "extra". I think this is fine... but it wouldn't be impossible to factor out just the code we need from the library.